lua/lazyvim/plugins/extras/lang/clojure.lua

```diff
@@ -0,0 +1,101 @@
+return {
+  recommended = function()
+    return LazyVim.extras.wants({
+      ft = { "clojure", "edn" },
+      root = { "project.clj", "deps.edn", "build.boot", "shadow-cljs.edn", "bb.edn" },
+    })
+  end,
+
+  -- Add Clojure & related to treesitter
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      opts.ensure_installed = opts.ensure_installed or {}
+      vim.list_extend(opts.ensure_installed, { "clojure" })
+    end,
+  },
```

Comment:
```suggestion
    opts = { ensure_installed = { "clojure" } },
```
With the new changes in `lazy.nvim` this is not needed any more. You can directly use `ensure_installed` without `vim.list_extend`.

Reply:
Fixed it already :)